docs/source/checks/vision/distribution/plot_new_labels.py

```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
+"""
+New Labels
+***************
+
+This notebooks provides an overview for using and understanding the New Labels check.
+
+**Structure:**
+
+* `How the check works <#How-the-check-works>`__
+* `Run the check <#run-the-check>`__
+
+How the check works
+========================
+In this check we count the frequency of each class id in the test set then check which of them
+do not apper in the training set.
+"""
+
+# %%
+# Imports
```

Comment:
Don't think that imports deserves it's own section. Should be done in start of Run the Check section.